src/realm/sync/client.cpp

```diff
@@ -228,6 +232,8 @@ class SessionWrapper final : public util::AtomicRefCountBase, DB::CommitListener
     // Must only be accessed from the event loop thread.
     SessionImpl* m_sess = nullptr;
 
+    // @}
```

Comment:
Maybe we can move these fields to some other class, and make it fully threadsafe? This is a bit unsettling that only part of this class is supposed to be accessed from different threads. It's error prone.